File: Foundations/Interpret/interpret.java
public int interpret(int value, String[] commands, int[] args) {
	int operandLength = commands.length;
	int i = 0;
	while (i < operandLength) {
		switch (commands[i]) {
			case "+":
				value += args[i];
				break;
			case "-":
				value -= args[i];
				break;
			case "*":
				value *= args[i];
				break;
			default:
				return -1;
		}
		i++;
	}

	return value;
}

File: Foundations/Interpret/interpret.py
def interpret(value, commands, args):
	i = 0
	length = len(commands);
	while (i < length):
		if ("+" == commands[i]):
			value += args[i]
		elif ("-" == commands[i]):
			value -= args[i]
		elif ("*" == commands[i]):
			value *= args[i]
		else:
			return -1
		i+=1;
	return value;
